### DISCUSSION TITLE
Week 2

### DISCUSSION THREAD

**Discussion by Stev3eM** [SE-TINF22B6/CookHub#25] [Show and tell]
Hello backpackers,

this is our weekly discussion-blog to keep you up to date within our project progression.
For the second week we have a few updates and basic infos to release:

1. Our Team
At first we want to introduce ourself to all of you:

<img width="309" alt="image" src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/42705ada-6f42-4a94-b4ef-028b7f34f693">

2. Product Vision
CookHub is intended to be an online recipe book that can store its own recipes in a database, suggest recipes via the OpenAI API, can save your own created recipes, and create a shopping list from an existing recipe. In addition, the OpenAI API allows a recipe to be turned into an adventure by wrapping the cooking instructions in an automatically generated adventure story to turn simple cooking into a real experience.
   
3. Logo Prototype
We pruoudly introduce our first logo portotype which could be replaced in the future:<br>
<img width="484" alt="CookHub_Logo_final2" src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/6e82f9e7-2e78-40c9-8189-391b37ec22a1">

4. Technolgies
We decided to use the following technolgies so far: 
React, .Net, C#, JS/TS, TMETRIC, GitHub

_(! Could change or extend in the future)_

5. Our team agreed on some project rules
✅ Sprint length = two weeks
✅ When/how long is your sprint meeting? {review + retrospective + plan} = {~20, ~15, ~90}
✅ How often is your daily? = On demand in addition to our weekly

6. Setup our scrum platform
• We initialized our project in GitHub: 
✅Defined sprints, ✅Defined categories, ✅requirement analysis, ❌ design (in planning), ✅implementation of our project to repository, ✅environment setup (React, .Net, C#, JS/TS, TMETRIC, GitHub), ✅documentation
• Create issue templates: ✅Epic & User Story & Tasks, ✅Setup Backlog & view headers, ✅setup fo report your hours by time spent per person & hours by time spent per category = TMETRIC AddOn, ✅Fill in your backlog (At least 3 realistic epics per team) & (At least 5 realistic user stories per team) & (At least 10 tasks per team), ✅mark categories, ✅assign each task to a team member, ✅define criteria

7. Run the first Sprint meeting
✅ What to be done in your first Sprint (all issues selected)
✅ Define your sprint backlog (several categories/tasks created)
✅ Everyone pulls own tasks in the first sprint in Kanban (Todo column)
✅setup API development environment 

8. "Create MockUp for Landing Page#17" via Figma:

![image](https://github.com/SE-TINF22B6/CookHub/assets/113697513/9e7bbf26-015a-4791-bb94-356ba798c309)

9. Lastly we provide the first screenshot of our Kanban:

![image](https://github.com/SE-TINF22B6/CookHub/assets/113697513/b4bea34a-8b10-4931-8a5c-2f77834991d6)

**Comment by giomskii** [comment]
Really good overview and progression over one week. Appreciate your mockup and logo design as well as the structure of your Github tasks/issues 👍

**Comment by AndresMasis** [comment]
I congratulate you. It is evident that you have put a lot of effort in your product since the beginning. The project seems to be well organized and with a clear development path. Wish you success.
